c10/core/Allocator.h

```diff
@@ -74,7 +74,7 @@ class C10_API DataPtr {
    * However, in some cases, you know /exactly/ what the deleter
    * is, and you have a new deleter that manually wraps
    * the old one.  In this case, you can safely swap the deleter
-   * after asserting that the deleters line up.
+   * after asserting that the deleters line-up.
```

Comment:
Hyphenating this is incorrect because then it would be an adjective. This "line up" is intended to be a verb.

Reply:
Hmm, looks like it's also valid as one word: https://www.merriam-webster.com/dictionary/lineup
> an alignment (as in entertainment or politics) of persons or things having a common purpose, distinction, or bond